[2024-02-19T09:27:50.507+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_project.Load_song_dim_table manual__2024-02-19T09:25:05.177058+00:00 [queued]>
[2024-02-19T09:27:50.514+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_project.Load_song_dim_table manual__2024-02-19T09:25:05.177058+00:00 [queued]>
[2024-02-19T09:27:50.515+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 4
[2024-02-19T09:27:50.523+0000] {taskinstance.py:2192} INFO - Executing <Task(LoadDimensionOperator): Load_song_dim_table> on 2024-02-19 09:25:05.177058+00:00
[2024-02-19T09:27:50.527+0000] {standard_task_runner.py:60} INFO - Started process 703 to run task
[2024-02-19T09:27:50.531+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'final_project', 'Load_song_dim_table', 'manual__2024-02-19T09:25:05.177058+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/final_project.py', '--cfg-path', '/tmp/tmpdmr7_z_5']
[2024-02-19T09:27:50.534+0000] {standard_task_runner.py:88} INFO - Job 162: Subtask Load_song_dim_table
[2024-02-19T09:27:50.565+0000] {task_command.py:423} INFO - Running <TaskInstance: final_project.Load_song_dim_table manual__2024-02-19T09:25:05.177058+00:00 [running]> on host b8bde2aaf906
[2024-02-19T09:27:50.607+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='udacity' AIRFLOW_CTX_DAG_ID='final_project' AIRFLOW_CTX_TASK_ID='Load_song_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-19T09:25:05.177058+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-19T09:25:05.177058+00:00'
[2024-02-19T09:27:50.612+0000] {base.py:83} INFO - Using connection ID 'aws_credentials' for task execution.
[2024-02-19T09:27:50.613+0000] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='aws_credentials', conn_type='aws') credentials retrieved from login and password.
[2024-02-19T09:27:50.619+0000] {load_dimension.py:36} INFO - dim.dim_songs is being inserted
[2024-02-19T09:27:50.623+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-19T09:27:56.747+0000] {sql.py:432} INFO - Running statement: delete from dim.dim_songs, parameters: None
[2024-02-19T09:27:57.833+0000] {sql.py:441} INFO - Rows affected: 14896
[2024-02-19T09:27:58.892+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-19T09:28:02.602+0000] {sql.py:432} INFO - Running statement: 
    insert into dim.dim_songs (song_id, title, artist_id, year, duration)
        SELECT distinct song_id, title, artist_id, year, duration
        FROM staging.staging_songs
    , parameters: None
[2024-02-19T09:28:04.554+0000] {sql.py:441} INFO - Rows affected: 14896
[2024-02-19T09:28:05.642+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=final_project, task_id=Load_song_dim_table, execution_date=20240219T092505, start_date=20240219T092750, end_date=20240219T092805
[2024-02-19T09:28:05.654+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-19T09:28:05.665+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
